test/distributed/elastic/agent/server/test/local_elastic_agent_test.py

```diff
@@ -32,8 +33,9 @@
 )
 from torch.distributed.elastic.agent.server.local_elastic_agent import (
     LocalElasticAgent,
-    TORCHELASTIC_TIMER_FILE,
+    TORCHELASTIC_ENABLE_FILE_TIMER,
 )
+from torch.distributed.elastic.events.fb.scuba import ScubaLogHandler
```

Comment:
I'm surprised we don't see any failures from this. Are torch.elastic tests even running in our OSS CI? How can we be importing from internal fb-only code in our open source mirror? cc @d4l3k

Reply:
@H-Huang I'm not 100% sure -- how are are the builds configured in OSS CI? / how do I check? 

@kiukchung would probably have the best context here

Reply:
I moved the scuba test to a FB internal test module.

Reply:
the only tests we "blocklist" are defined [here](https://github.com/pytorch/pytorch/blob/master/test/run_test.py#L95), which doesn't list `distributed.elastic.agent.server.test.local_elastic_agent_test`. Did the CI that was triggered for this PR not run `local_elastic_agent_test.py`?